examples/fundamentals.test.js

```diff
@@ -4418,6 +4418,47 @@ describe('Fundamentals', function() {
       arr.findIndex(n => Number.isNaN(n)); // 4
       
     });
+    it('let', function() {
+      let x = 1;
+        if (x === 1) {
+          let x = 2;
+          if (x === 2) {
+            let x = 3;
+            console.log(x);
+            // prints 3
+          }
+          console.log(x);
+          // prints 2
+        }
+
+      console.log(x);
+      // prints 1
+    });
+    it('var', function() {
+      var x = 1;
```

Comment:
It would be better for this to line up exactly with the `let` example. Hoisting is neat but it would be better to explain it in a separate example.